frontend/src/Components/ControlPanelFeatureGate/ControlPanelFeatureGate.tsx

```diff
@@ -0,0 +1,14 @@
+import { Navigate } from 'react-router-dom';
+import { isControlPanelFeatureEnabled } from '~/config/controlPanelFeatures';
+import { firstEnabledAdminPath } from '~/config/controlPanelRouting';
+import type { ControlPanelFeature } from '~/types';
+
+export function controlPanelFeatureGate({
```

Comment:
React-komponenter skal starte med stor bokstav (trenger å oppdateres der hvor controlPanelFeatureGate importes også)

```suggestion
export function ControlPanelFeatureGate({
```